dags/inference/jetstream_inference_e2e.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A DAG to run JetStream inference E2E test."""
+
+import datetime
+from airflow import models
+from dags.vm_resource import TpuVersion, Zone, Project, V5_NETWORKS, V5E_SUBNETWORKS, RuntimeVersion
+from dags.inference.configs import maxtext_inference_gce_config
+from dags.multipod.configs.common import SetupMode
+
+"""A JetStream inference E2E test DAG.
+
+Usage:
+gcloud composer environments run ml-automation-solutions \
+  --project=cloud-ml-auto-solutions \
+  --location=us-central1 dags trigger \
+  -- \
+  jetstream_e2e_inference
+
+"""
```

Comment:
Should we add a comment about how this does not run nightly, and that this is meant for manual trigger only?

Reply:
It's using JAX nightly, updated.